src/FSharpSource.targets

```diff
@@ -185,6 +185,7 @@
     <DefineConstants>$(DefineConstants);NETSTANDARD1_6</DefineConstants>
     <DefineConstants>$(DefineConstants);NETCOREAPP1_0</DefineConstants>
     <OtherFlags>$(OtherFlags) --simpleresolution</OtherFlags>
+    <TargetProfile>netcore</TargetProfile>
```

Comment:
This is all internal build engineering stuff trying to clear up the legacy, Right now we use MSBuild and Microsoft.DotNet.BuildTools in this repo to build .NET Core stuff.  No one else does that except corefx, it's just different

Enrico's stuff is orthogonal - when it's real it will help us to converge